Restructure Node SDK docs. (#5710)
getsentry/sentry-docs: src/platform-includes/configuration/integrations/pluggable-integrations/node.mdx

> Pluggable integrations are integrations that can be additionally enabled, to provide specific features. They're documented so you can see what they do and that they can be enabled. To enable pluggable integrations, provide a new instance with your config to the `integrations` option. For example: `integrations: [new Sentry.Integrations.Modules()]`.

Reviewer: ```suggestion
Pluggable integrations are integrations that can be additionally enabled to provide specific features. They're documented so you can understand what they do and enable them, if needed. To enable pluggable integrations, provide a new instance with your config to the `integrations` option. For example: `integrations: [new Sentry.Integrations.Modules()]`.
```

## Core

### CaptureConsole

_Import name: `Sentry.Integrations.CaptureConsole`_

This integration captures all `Console API` calls and redirects them to Sentry using `captureMessage` call.

> It then retriggers to preserve default native behavior.

Reviewer: ```suggestion
It then re-triggers to preserve default native behavior.
```

Available options:

```javascript
{
  levels: string[]; // an array of methods that should be captured, defaults to ['log', 'info', 'warn', 'error', 'debug', 'assert']
}
```

### Dedupe

_Import name: `Sentry.Integrations.Dedupe`_

> This integration deduplicates certain events. It can be helpful if you are receiving many duplicate errors. Be aware that we will only compare stack traces and fingerprints.

Reviewer: ```suggestion
This integration deduplicates certain events. It can be helpful if you're receiving many duplicate errors. Be aware that we will only compare stack traces and fingerprints.
```

### Debug

_Import name: `Sentry.Integrations.Debug`_

This integration allows you to quickly inspect the content of the processed event that will be passed to `beforeSend` and effectively send it to Sentry.

Available options:

```javascript
{
  debugger: boolean; // trigger DevTools debugger instead of using console.log
  // stringify event before passing it to console.log
  stringify: boolean;
}
```

### ExtraErrorData

_Import name: `Sentry.Integrations.ExtraErrorData`_

> This integration extracts all non-native attributes from the Error object and attaches them to the event as the `extra` data.

Reviewer: ```suggestion
This integration extracts all non-native attributes from the error object and attaches them to the event as the `extra` data.
```

Available options:

```javascript
{
  // limit of how deep the object serializer should go. Anything deeper than limit will
  // be replaced with standard Node.js REPL notation of [Object], [Array], [Function] or
  // a primitive value. Defaults to 3.
  depth: number;
}
```

### RewriteFrames

_Import name: `Sentry.Integrations.RewriteFrames`_

This integration allows you to apply a transformation to each frame of the stack trace. In the streamlined scenario, it can be used to change the name of the file frame it originates from, or it can be fed with an iterated function to apply any arbitrary transformation.

> On Windows machines, you have to use Unix paths and skip the volume letter in `root` option in order to make it work.
> For example `C:\\Program Files\\Apache\\www` won't work, however, `/Program Files/Apache/www` will.

Reviewer: ```suggestion
On Windows machines, you have to use Unix paths and skip the volume letter in the `root` option to make it work.
For example, `C:\\Program Files\\Apache\\www` won't work, however, `/Program Files/Apache/www` will.
```

<PlatformContent includePath="configuration/rewrite-frames" />

> ## Node specific

Reviewer: ```suggestion
## Node-specific
```
